metricbeat/module/jolokia/_meta/config.yml

```diff
@@ -0,0 +1,7 @@
+- module: jolokia
```

Comment:
Currently we commend out the config by default so it is not enabled. This might change in the near future: https://github.com/elastic/beats/blob/master/metricbeat/module/kafka/_meta/config.yml